docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ FROM socrata/java
 ENV SERVER_NAME region-coder
 ENV SERVICE_PORT 2021
 EXPOSE 2021
+ENV JMX_PORT 9999
+ENV JMX_RMI_PORT 9998
```

Comment:
So is the idea every process will have their own separate ports mapped to the same port on the host?  If so we should try to (1) see if we can use only one port as per http://stackoverflow.com/questions/20884353/why-java-opens-3-ports-when-jmx-is-configured and (2) put the port in the rage that the app port is for profit.

If not, we need to tell the port mapping to JMX/RMI somehow?